src/worker.cc

```diff
@@ -138,6 +138,8 @@ const std::unordered_map<unsigned, valhalla::valhalla_exception_t> error_codes{
     {445, {445, "Shape match algorithm specification in api request is incorrect. Please see documentation for valid shape_match input.", 400, HTTP_400, OSRM_INVALID_URL, "wrong_match_type"}},
     {499, {499, "Unknown", 400, HTTP_400, OSRM_INVALID_URL, "unknown"}},
     {503, {503, "Leg count mismatch", 400, HTTP_400, OSRM_INVALID_URL, "wrong_number_of_legs"}},
+    {504, {504, "This service does not support GeoTIFF serialization.", 400, HTTP_400, OSRM_INVALID_VALUE, "wrong_number_of_legs"}},
```

Comment:
```suggestion
    {504, {504, "This service does not support GeoTIFF serialization.", 400, HTTP_400, OSRM_INVALID_VALUE, "unknown"}},
```
? not sure how/if that maps to any osrm error description

Reply:
Yeah I didn't get what these errors have to do with OSRM anyway, probably compatibility with clients for mapbox?

Reply:
Jep that’s right. Same as the whole “osrm” format business for the response.

Reply:
looking at all comments this seems to be the only one left where we need to do smth. without going into osrm error codes, IMO "unknown" should never be totally wrong :) at least `wrong_number_of_legs` is wrong